security/design/jwt.go

```diff
@@ -0,0 +1,34 @@
+package design
+
+import (
+	. "github.com/goadesign/goa/design"
+	. "github.com/goadesign/goa/design/apidsl"
+)
+
+// JWT defines a security scheme using JWT.  The scheme uses the "Authorization" header to lookup
+// the token.  It also defines a required scope "api".
+var JWT = JWTSecurity("jwt", func() {
+	Header("Authorization")
+	Scope("api", "API access")
+})
+
+var _ = Resource("JWTEndpoints", func() {
+	Description("This resource uses JWT to secure its endpoints")
+	DefaultMedia(SuccessMedia)
+
+	Security(JWT)
+
+	Action("secured", func() {
+		Description("This action is secured with the jwt scheme")
+		Routing(GET("/jwt"))
```

Comment:
I'd add a `Scope("api")` in here perhaps.. so that we use the `"api"` scope defined on line 12.